tests/qauto/homework1.spec.js

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from "@playwright/test";
+
+
+test.describe('Registration form test', () => {
+
+    test.beforeEach(async ({page})=>{
+        await page.goto("https://qauto.forstudy.space/");
+        await page.locator(".btn-primary").click();
+        await expect(page.locator(".modal-content")).toBeVisible();
+    })
+
+  test('Positive', async ({ page }) => {
+    await page.locator("#signupName").fill("Justin");
+    await page.locator("#signupLastName").fill("Potter");
+    await page.locator("#signupEmail").fill("dns-jp@test.com");
+    await page.locator("#signupPassword").fill("qwertY@123");
+    await page.locator("#signupRepeatPassword").fill("qwertY@123");
```

Comment:
do not hardcode values

Reply:
after you have it moved to variables you can get rid of expectedName, expectedLastName etc.